app_test.py

```diff
@@ -73,6 +74,77 @@ def test_update_note_with_invalid_token(self):
         print("test_get_all_notes_with_invalid_token Response - " + str(response.data))
         self.assertEqual(response.status_code, 403)
 
+    def test_integration_register_post_and_update(self):
+        _name = "Ajit S"
+        _id = "ajit1@ajit.com"
+
+        # Registration phase
+        reg_resp = self.app.post("/register", json={"id": _id, "name": _name})
+        r_dict = json.loads(reg_resp.get_json())
+        _token = r_dict["token"]
+
+        self.assertEqual(reg_resp.status_code, 200)
+
+        headers = {"Content-Type": "application/json", "token": _token}
+        # Posting phase
+        note_payload = {"note": "test note", "is_public": False}
+        add_resp = self.app.post("/add_note", headers=headers, json=note_payload)
+        note_response_payload = json.loads(add_resp.get_json())
+
+        _note_id = note_response_payload["note_id"]
+
+        self.assertEqual(add_resp.status_code, 200)
+        self.assertEqual(note_response_payload["note"], note_payload["note"])
+        self.assertEqual(note_response_payload["owner_id"], _id)
+
+        # Updating phase
+        update_payload = {"note_id": _note_id, "note": "updated test note"}
+        upd_resp = self.app.post("/update_note", headers=headers, json=update_payload)
+        update_response_payload = json.loads(upd_resp.get_json())
+        print(update_response_payload)
+
+        self.assertEqual(upd_resp.status_code, 200)
+        self.assertEqual(update_response_payload["status"], "success")
+
+    def test_integration_register_post_set_public(self):
+        _name = "Roman P"
+        _id = "roman@roman.com"
+
+        # Registration phase
+        reg_resp = self.app.post("/register", json={"id": _id, "name": _name})
+        print(reg_resp.get_json())
+        r_dict = json.loads(reg_resp.get_json())
+        _token = r_dict["token"]
+
+        headers = {"Content-Type": "application/json", "token": _token}
+
+        # Posting phase
+        note_payload = {"note": "test note", "is_public": False}
+        add_resp = self.app.post("/add_note", headers=headers, json=note_payload)
+        note_response_payload = json.loads(add_resp.get_json())
+
+        _note_id = note_response_payload["note_id"]
+
+        self.assertEqual(add_resp.status_code, 200)
+        self.assertEqual(note_response_payload["note"], note_payload["note"])
+        self.assertEqual(note_response_payload["owner_id"], _id)
+
+        # Updating phase
+        update_payload = {"note_id": _note_id, "is_public": True}
+        upd_resp = self.app.post("/update_note", headers=headers, json=update_payload)
+        update_response_payload = json.loads(upd_resp.get_json())
+        print(update_response_payload)
```

Comment:
nit: remove or comment out this print statement

Reply:
Taken care of. Thanks!